lib/tasks/autotest.rake

```diff
@@ -127,6 +129,24 @@ class AutotestSetup
     )
   end
 
+  def create_submission_files(transaction, glob)
```

Comment:
Same here, this recursive definition is designed to support the `src` folder in the rust submission.